tests/unit/amo/components/TestAddAddonToCollection.js

```diff
@@ -90,35 +88,36 @@ describe(__filename, () => {
 
   describe('fetching user collections', () => {
     it('fetches user collections on first render', () => {
-      const username = 'some-user';
-      dispatchSignInActions({ store, userProps: { username } });
+      const userId = 345;
+      dispatchSignInActions({ store, userId });
       const dispatchSpy = sinon.spy(store, 'dispatch');
+
       const root = render();
 
       sinon.assert.calledWith(
         dispatchSpy,
         fetchUserCollections({
           errorHandlerId: root.instance().props.errorHandler.id,
-          userId: username,
+          userId: String(userId),
```

Comment:
Should this really be a string? Is that what we want our types to be? I'm guessing it's based on the API, but would it make more sense to use numbers internally where we know we expect numbers, and change them into strings in the API code if that's what the API expects?